src/aria-snapshots/dropdown-test-aria-ts-multiple-false-open-true-.yml

```diff
@@ -1,6 +1,6 @@
-- text: Label
-- button "Label" [expanded]
+- text: Label One, One
```

Comment:
Not sure why there's two of "One" in the tree. Only one of them is read by screenreaders.